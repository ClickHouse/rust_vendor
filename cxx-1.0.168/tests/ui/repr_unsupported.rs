#[cxx::bridge]
mod ffi {
    #[repr(align(2))]
    enum EnumAlign {
        A,
    }

    #[repr(i64)]
    struct StructInt {
        i: i32,
    }

    #[repr(align(1 << 10))]
    struct StructExpr {
        i: i32,
    }
}

fn main() {}
