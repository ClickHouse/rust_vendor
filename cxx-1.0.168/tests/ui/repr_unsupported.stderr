error: C++ does not support custom alignment on an enum
 --> tests/ui/repr_unsupported.rs:3:18
  |
3 |     #[repr(align(2))]
  |                  ^

error: unsupported alignment on a struct
 --> tests/ui/repr_unsupported.rs:8:12
  |
8 |     #[repr(i64)]
  |            ^^^

error: invalid repr(align) attribute: an arithmetic expression is not supported
  --> tests/ui/repr_unsupported.rs:13:18
   |
13 |     #[repr(align(1 << 10))]
   |                  ^^^^^^^
